static/js/containers/CreatePostPage.js

```diff
@@ -57,14 +58,19 @@ class CreatePostPage extends React.Component<*, void> {
 
   onUpdate = (e: Object) => {
     const { dispatch } = this.props
+    const { name, value } = e.target
+
     dispatch(
       actions.forms.formUpdate({
         ...CREATE_POST_PAYLOAD,
         value: {
-          [e.target.name]: e.target.value
+          [name]: value
         }
       })
     )
+    if (name === "url") {
+      dispatch(actions.embedly.get(value))
```

Comment:
I think we should consider debouncing this somehow. Granted most users will copy/paste an address, but this can cause a lot of extraneous requests since it'll trigger a request for every keypress.

Reply:
Also, this should not query embedly if the value is empty. To reproduce, enter a link and then clear the field.

Reply:
yup, that makes sense >_<